playground/app.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+	<div>
+		<NuxtLayout>
+      <NuxtPage />
+		</NuxtLayout>
+	</div>
+</template>
```

Comment:
could we lint the project so we have consistent spacing? (looks like some files have tabs, some spaces)...